src/main/java/org/folio/search/repository/SearchRepository.java

```diff
@@ -1,24 +1,62 @@
 package org.folio.search.repository;
 
+import static org.folio.search.utils.SearchUtils.getElasticsearchIndexName;
+import static org.folio.search.utils.SearchUtils.performExceptionalOperation;
+
+import java.util.Arrays;
+import java.util.List;
+import java.util.Map;
+import java.util.stream.Collectors;
+import lombok.RequiredArgsConstructor;
+import org.elasticsearch.action.search.SearchRequest;
+import org.elasticsearch.action.search.SearchResponse;
+import org.elasticsearch.client.RequestOptions;
+import org.elasticsearch.client.RestHighLevelClient;
 import org.elasticsearch.index.query.QueryBuilder;
+import org.elasticsearch.search.SearchHit;
+import org.elasticsearch.search.builder.SearchSourceBuilder;
 import org.folio.search.model.rest.response.SearchResult;
+import org.folio.search.model.service.CqlSearchRequest;
 import org.springframework.stereotype.Repository;
 
 /**
  * Search resource repository with set of operation to perform search operations.
  */
 @Repository
+@RequiredArgsConstructor
```

Comment:
Consider to do the same for `IndexRepository.java`

Reply:
fixed